kie-dmn-feel/src/main/java/org/kie/dmn/feel/lang/FEELAccessor.java

```diff
@@ -0,0 +1,15 @@
+package org.kie.dmn.feel.lang;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Retention(RUNTIME)
+@Target(ElementType.METHOD)
+public @interface FEELAccessor {
```

Comment:
Can I suggest a rename to @FEELProperty? seems more intuitive for users than @FEELAccessor. Just a suggestion.